third_party/chromium/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
why are we duplicating the chromium license? This seems like a bad idea because it could change also seems unnecessary

Reply:
You're right, I think I can get rid of this one via a "License File" field.